packages/client/hmi-client/src/components/model/tera-model.vue

```diff
@@ -181,11 +181,17 @@ async function fetchConfigurations() {
 
 		// Ensure that we always have a "default config" model configuration
 		if (
-			isEmpty(tempConfigurations) ||
-			!tempConfigurations.find((d) => d.name === 'Default config')
+			(isEmpty(tempConfigurations) ||
+				!tempConfigurations.find((d) => d.name === 'Default config')) &&
+			!isModelEmpty(model.value)
 		) {
 			await addDefaultConfiguration(model.value);
-			tempConfigurations = await getModelConfigurations(model.value.id);
+			setTimeout(async () => {
+				// elastic search might still not update in time
+				tempConfigurations = await getModelConfigurations(model.value?.id!);
+				modelConfigurations.value = tempConfigurations;
+			}, 800);
+			return;
```

Comment:
wrapping this in a timeout...it seems that model configuration aren't updated immediately